### PR TITLE
Feature/javascript features

### PR DESCRIPTION
**Pull request by Harrisong-Gutierrez**
Feature/javascript features

### CODE REVIEW

File: src/components/footer.html [hunk 1]
```diff
@@ -56,4 +39,21 @@
             </div>
             <img class="Footer-left-gif" src="/public/gif1.gif" alt="fotter-gif">
         </div>
+    </div>
+    <div class="w-100 d-flex align-items-center justify-content-center Footer-left-privacy">
+        <p>
+            Copyright ©
+            <script>document.write(new Date().getFullYear());</script>2023 All rights reserved | This
```
Comment (robSamuel) [inline src/components/footer.html]: Delete **2023**, you're actually getting the year from JS

File: src/components/footer.html [hunk 2]
```diff
@@ -56,4 +39,21 @@
...
+            template is made with <i class="fa fa-heart-o" aria-hidden="true"></i> by <a
+                href="https://www.smbssolutions.com/" target="_blank">SMBS</a>
+        </p>
+
+        <div class="Footer-left-icons">
```
Comment (robSamuel) [inline src/components/footer.html:51]: Add all the links for the anchor tags

**Review by robSamuel — CHANGES_REQUESTED** [review]

File: src/components/footer.html (hunk 2 above)
Comment (Harrisong-Gutierrez) [reply src/components/footer.html:51]: done

File: src/components/footer.html (hunk 1 above)
Comment (Harrisong-Gutierrez) [reply src/components/footer.html]: done

**Review by robSamuel — APPROVED** [review]

**Pull request merged**